README.md

```diff
@@ -54,27 +63,34 @@ personal access token on your local OSF instance ([here](http://localhost:5000/s
 
 #### Create a local settings file
 
-To do this:
+If for some reason you don't have a config/local.yml you can generate one. To do this:
 ```bash
-ember g ember-osf-settings `echo $HOSTNAME`
+ember g ember-osf `echo $HOSTNAME`
```

Comment:
Very minor future nitpick: I really like the shorthand commands like `ember g` myself... but when we are explaining this to people not familiar with ember, we should spell out the entire command explicitly. It's a small bit of polish that helps make a new tool more approachable.

`ember generate ...` vs `ember g`, `ember server` vs `ember s`, etc.

The full names are the ones most commonly explained in documentation, and aliases are one more layer that people have to learn about before they can get started.